src/Forms/FormTextarea.js

```diff
@@ -37,10 +56,19 @@ FormTextarea.propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
     compact: PropTypes.bool,
+    counter: PropTypes.number,
```

Comment:
HTML `textarea` elements have existing support for the `maxLength` prop.  

We could re-use that language as it may be more familiar to consumers.